http/get_simple/java/server/src/main/java/com/example/ArrowHttpServer.java

```diff
@@ -0,0 +1,131 @@
+import org.apache.arrow.memory.BufferAllocator;
```

Comment:
just curious should we include licensing here?

Reply:
Ah yeah, thanks for the reminder! Done. I also added the license comment at the top of all the other code files where it was missing in #5.